packages/compute/src/vm.js

```diff
@@ -407,6 +407,74 @@ VM.prototype.reset = function(callback) {
 };
 
 /**
+ * Set the machine type for this instance, **stopping and restarting the VM as
+ * necessary**.
+ *
+ * For a list of the standard, high-memory, and high-CPU machines you may choose
+ * from, see
+ * [Predefined machine types]{@link https://cloud.google.com/compute/docs/machine-types#predefined_machine_types}.
+ *
+ * In order to change the machine type, the VM must not be running. This method
+ * will automatically stop the VM if it is running before changing the machine
+ * type. After it is sucessfully changed, the VM will be started.
```

Comment:
WDYT about adding a `start: false` option? By default we restart the vm, however they can optionally leave it off if they specify?